tools/cldr-code/src/main/java/org/unicode/cldr/util/XPathParts.java

```diff
@@ -34,6 +34,11 @@
  * //ldml/characters/exemplarCharacters[@type="auxiliary"] and a list of Element objects that depend
  * on xPath. Each Element object has an "element" string such as "ldml", "characters", or
  * "exemplarCharacters", plus attributes such as a Map from key "type" to value "auxiliary".
+ *
+ * <p>Caches values for fast lookup.
+ *
+ * <p>For use at static init time, or anywhere a simpler API can be used, see {@link
```

Comment:
A downside of SimpleXPathParts is that it doesn't cache, right? That should be mentioned here also, because it is a downside to using SimpleXPathParts for repeated usage.

Reply:
Yes. This should say, static init time _where XPaths that are going to be read only once are used_.  Especially supplemental data, where all XPaths are unique and caching doesn't help.